tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/DebugLogger.cs

```diff
@@ -89,7 +89,7 @@ public static void LogDebug(string details)
             }
             else
             {
-                Debug.WriteLine(details);
+                System.Console.WriteLine(details);
```

Comment:
Discovered that `debug.writeline` doesn't show up by default on command line calling the `test-proxy` tool. Given that, adjusted to console out directly.